samples/meetings-events/csharp/MeetingEvents/Bots/ActivityBot.cs

```diff
@@ -23,6 +25,70 @@ public ActivityBot(ConversationState conversationState)
             _conversationState = conversationState;
         }
 
+        /// <summary>
+        /// Activity Handler for Meeting Participant join event
+        /// </summary>
+        /// <param name="meeting"></param>
+        /// <param name="turnContext"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        protected override async Task OnTeamsMeetingParticipantsJoinAsync(MeetingParticipantsEventDetails meeting, ITurnContext<IEventActivity> turnContext, CancellationToken cancellationToken)
+        {
+            await turnContext.SendActivityAsync(MessageFactory.Attachment(createAdaptiveCardInvokeResponseAsync(meeting.Members[0].User.Name, " has joined the meeting.")));
```

Comment:
message should be  " joined the meeting." to match the other event which says " left the meeting."

Reply:
not a blocker to merge